pages/vi/vi-github-and-repositories.md

```diff
@@ -175,11 +175,11 @@ For more info on differences of merging vs. rebasing (and when to use which one)
 
 ## Summary of steps
 
-1. Clone your git \<username\> repository and open your terminal window to it by using ``cd``
+1. Clone your GitHub repository username.github.io and open your terminal window to it by using ``cd``
```

Comment:
The original sentence does not sounds right, could you fix it?

Reply:
I already replaced the original sentence "Clone your git \ repository" with "Clone your GitHub repository username.github.io". Could you remind me what this sentence should be written or what else need to be fixed?

Reply:
@YangLou1 I was referring to the latter part of the sentence, "open your terminal window to it by using cd"

Reply:
@xyb994 Thanks for reminding me. I have revised the latter part of the sentence based on the same insturction on the page.